xml/System.Buffers/ArrayBufferWriter`1.xml

```diff
@@ -87,8 +90,18 @@
       </Parameters>
       <Docs>
         <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Notifies <see cref="T:System.Buffers.IBufferWriter`1" /> that <paramref name="count" /> amount of data was written to the output <see cref="T:System.Span`1" />/<see cref="T:System.Memory`1" /></summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+You must request a new buffer after calling Advance to continue writing more data and cannot write to a previously acquired buffer.
```

Comment:
@rpetrusha @mairaw is this really a langword? Shouldn't it be a "see cref"?

Reply:
Since this is the method itself, I think we can do like this:

```suggestion
You must request a new buffer after calling `Advance` to continue writing more data and you cannot write to a previously acquired buffer.
```